packages/host/tests/integration/components/serialization-test.gts

```diff
@@ -4427,6 +4427,95 @@ module('Integration | serialization', function (hooks) {
     // this module checks the custom serialization and deserialization behaviour of base cards
     // which have custom serialize and deserialize
 
+    module('NumberCard', function () {
```

Comment:
Its because of the manual override of [deserialize] that these test are useful. To ensure the developer knows what hes doing